### ISSUE TITLE
Remove unused prometheus rules

### ISSUE THREAD

**Issue by sumo-drosiek** [SumoLogic/sumologic-kubernetes-collection#3182]
The following prometheus rules are no longer used by kubernetes application in sumologic, and therefore should be removed:

```

# cluster_quantile:apiserver_request_duration_seconds:histogram_quantile
# cluster_quantile:scheduler_binding_duration_seconds:histogram_quantile
# cluster_quantile:scheduler_framework_extension_point_duration_seconds:histogram_quantile
# cluster_quantile:scheduler_e2e_scheduling_duration_seconds:histogram_quantile
# cluster_quantile:scheduler_scheduling_algorithm_duration_seconds:histogram_quantile
# instance:node_filesystem_usage:sum
# instance:node_network_receive_bytes:rate:sum
# node:cluster_cpu_utilisation:ratio
# node:node_filesystem_avail:
# node:node_inodes_free:
# node:node_inodes_total:
# node:node_memory_utilisation_2:
# node:node_net_saturation:sum_irate
# node:node_net_utilisation:sum_irate
```

**Comment by sumo-drosiek** [comment]
It should be validated as some of them may be used to calculate other prometheus rules :sweat:

**Comment by sumo-drosiek** [comment]
List of used prometheus rules:

```
node_load1
node_load5
node_load15
:kube_pod_info_node_count:
:node_cpu_saturation_load1:
:node_cpu_utilisation:avg1m
:node_disk_saturation:avg_irate
:node_disk_utilisation:avg_irate
:node_memory_utilisation:
:node_net_saturation:sum_irate
:node_net_utilisation:sum_irate
node:cluster_memory_utilisation:ratio
node:node_cpu_saturation_load1:
node:node_cpu_utilisation:avg1m
node:node_disk_saturation:avg_irate
node:node_disk_utilisation:avg_irate
node:node_filesystem_usage:
node:node_memory_bytes_total:sum
node:node_memory_utilisation:
node:node_memory_utilisation:ratio
node:node_net_saturation:sum_irate
node:node_net_utilisation:sum_irate
node:node_num_cpu:sum
node_namespace_pod:kube_pod_info
```

**Comment by swiatekm-sumo** [comment]
@sumo-drosiek this was resolved in #3211, can we close it?
